templates/manx.html

```diff
@@ -22,89 +21,154 @@ <h2>Manx</h2>
-                            </div>
-                        </td>
-                        <td>
-                            <div class="select is-small">
-                                <select id="procedure-filter" onchange="showProcedure()">
-                                    <option value="" disabled selected>Select a procedure</option>
-                                </select>
-                            </div>
-                        </td>
-                    </tr>
-                </table>
-            </center>
+        <div class="is-flex is-flex-direction-row is-justify-content-space-around">
+            <div class="select is-small">
+                <select id="session-id"
+                        x-model="selectedSessionID"
+                        x-on:change="getTactics()">
+                    <option value="" disabled selected>Select a session</option>
+                    <template x-for="s in sessions" :key="s.id">
+                        <option x-bind:value="s.id" x-bind:data-paw="s.info"
+                                x-bind:data-platform="s.platform"
+                                x-bind:data-executor="s.executors[0]"
+                                x-text="s.id + ' - ' + s.info"></option>
+                    </template>
+                </select>
+            </div>
+            <div class="select is-small">
+                <select x-model="selectedTacticName" id="tactic-filter" x-on:change="getTechniques()">
+                    <option value="" disabled selected>Select a tactic</option>
+                    <template x-for="a in tactics" :key="a.ability_id">
+                        <option x-text="a.tactic"></option>
+                    </template>
+                </select>
+            </div>
+            <div class="select is-small">
+                <select x-model="selectedTechniqueID" x-on:change="getProcedures()">
+                    <option value="" disabled selected>Select a technique</option>
+                    <template x-for="t in techniques" :key="t.technique_id">
+                        <option x-bind:value="t.technique_id"
+                                x-text="t.technique_id + ' | ' + t.technique_name"></option>
+                    </template>
+                </select>
+            </div>
+            <div class="select is-small">
+                <select id="procedure-filter" x-model="selectedProcedureID" x-on:change="getProcedure();">
+                    <option value="" disabled selected>Select a procedure</option>
+                    <template x-for="p in procedures">
+                        <option x-text="p.name" x-bind:value="p.ability_id"></option>
+                    </template>
+                </select>
             </div>
-            <br>
-            <div id="xterminal"></div>
         </div>
+        <br>
+        <div style="visibility:hidden;position:absolute" id="xterminal-command" x-text="terminalCommand"></div>
+        <div id="xterminal"></div>
```

Comment:
The cursor layer on the terminal doesn't take up the full width of the terminal container:
![Screen Shot 2021-11-01 at 5 00 29 PM](https://user-images.githubusercontent.com/6034483/139741611-971adf56-995b-45aa-8a52-a2064753917b.png)

Reply:
Good point! The UI of the terminal doesn't look great and when I went to check it out, seems that it is set to be that specific width somewhere in xterm most likely because otherwise it stretches out the words

<img width="1158" alt="Screen Shot 2021-11-03 at 21 50 21" src="https://user-images.githubusercontent.com/32778440/140244133-7144edbe-9004-44b0-a7f0-5d9e6573b9ba.png">

Reply:
Looks like there's an [official addon](https://xtermjs.org/docs/api/addons/fit/) to the xterm js library that can fit the terminal to the width of the parent element. Maybe an event listener for window width can be set up too, so that the terminal width is responsive if the screen size changes.